@navikt/ds-css/layouts.css

```diff
@@ -27,9 +27,15 @@
   padding: var(--navds-layout-padding-small);
 }
 
+/* prettier-ignore */
 .navds-layout__section--sticky > * {
-  top: calc(var(--navds-grid-gutter-small) + var(--decorator-sticky-offset));
+  /* TEMP Solution until dekratøren has updated */
+  --navds-sidebar-sticky-offset: var(--decorator-sticky-offset);
```

Comment:
Kanskje `--navds-descorator-sticky-offset` er et bedre navn? Jeg tenker jo at den kan være nyttig flere steder enn for sidebars. Evt. `--navds-descorator-top-offset` får å gjøre den enda mer generell. Eller `--navds-header-offset`. 

Mange muligheter. :)